modules/ossm-rn-new-features.adoc

```diff
@@ -15,6 +15,181 @@ Module included in the following assemblies:
 
 This release adds improvements related to the following components and concepts.
 
+//Includes 2.6, 2.5.3, 2.4.9.
+
+//OSSM 2.6 Release Day tentatively scheduled for Wednesday, July 10, 2024 --> delayed. New estimated release date: July 24, 2024.
+
+//This release will include new features such as OTEL+Tempo, Gateway API, and perhaps some others.
+
+[id="new-features-ossm-2-6"]
+== New Features {SMProductName} version 2.6
```

Comment:
I think this should be "New features...", which would match all of the previous "New features" sections. If so, note that there are additional subsection titles that also use "Features" instead of "features", so it would be good to check all of them.

Reply:
Good catch! Thank you!